### ISSUE TITLE
High "cpu_used_percentage" spikes since version v2023.2.2

### ISSUE THREAD

**Issue by Baxxy13** [richibrics/IoTuring#52]
Hi,
after upgrading from v2023.2.1 to v2023.2.2 i see high "cpu_used_percentage" spikes in the mqtt data.

I can see this on my 2 systems: 
RapberryPi 3B+:
```
~# uname -r
5.15.84-v8+
~# cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian 
```
Mini-PC (Celeron J4125) with Proxmox:
```
~# uname -r
6.0.15-edge
~# cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
```

When i check the systems locally with htop, i can see that IoTuring sometimes consumes up to 100% cpu.

"cpu_used_percentage" mqtt-data from my RaspberryPi:
section A: v2023.2.2
section B: v2023.2.1
![ioturing_cpu_spikes_B](https://user-images.githubusercontent.com/54898249/217773477-9f3d8ad1-b6db-4a01-a4f7-5f94ce4f77d2.JPG)

**Comment by richibrics** [comment]
Hi, thank you for the finding, could you try with the new branch "52-high-cpu_used_percentage-spikes-since-version-v202322" ?

Here you can find the pull request: #53

**Comment by Baxxy13** [comment]
Nice, this fixes the spikes as i can see so far.

Everything ist normal now, like in v2023.2.1.

Thanks!

**Comment by richibrics** [comment]
Nice !

#53 can be closed

**Comment by richibrics** [comment]
Fixed in 2022.2.3
